lib/Monads/trace/Trace_Det.thy

```diff
@@ -0,0 +1,74 @@
+(*
+ * Copyright 2023, Proofcraft Pty Ltd
```

Comment:
I'm a bit hazy on what we should do when a theory is new but contains only a subset of lines from another file. Should we be trying to pull in copyrights from the source file as well? @lsf37

Reply:
Thanks for spotting that, I think that's some copy paste mistakes that I made, since the copyrights for the new files aren't consistent. My intention was for all of the new files to inherit the copyrights from the files that I was copying the content from.